pkg/spec/createconfig.go

```diff
@@ -315,8 +315,21 @@ func (c *CreateConfig) GetTmpfsMounts() []spec.Mount {
 	return m
 }
 
+func createExitCommand(runtime *libpod.Runtime) []string {
+	config := runtime.GetConfig()
+
+	command := []string{os.Args[0],
```

Comment:
Should we be including anything else from the storage config in here

Reply:
I was looking for stuff that would effect the cleanup code, and did not see anything.  
log-level?
Here are the global options:
```
"cgroup-manager",     N Not running containers
"cni-config-dir",           N Not running containers
"config, c",                  M But i don't think so
"conmon",                  N Not running containers
"cpu-profile",              N Not running containers
"default-mounts-file",  N Not running containers
"hooks-dir-path",         N: Not running containers.
"log-level",                  M: Maybe
"storage-driver, s",       Y: We have this one.
"storage-opt",              M: Maybe
"root",                          Y: We have this one.
"tmpdir",                      N: Don't think so
"runroot"                     N: We have this one.
"runtime"                     N: We are not running containers
"storage-driver, s"       Y: Have this one.
"storage-opt"               M: I don't think we need this.
```

Reply:
We need `cgroup-manager` set to clean up CGroups. Might be a good idea to set tmpdir, too.

Reply:
Added